templates/tracking-script-wrapper.php

```diff
@@ -0,0 +1,11 @@
+<figure class="op-tracker">
+	<iframe>
+		<?php
+		if ( is_callable( $script ) ) {
+			call_user_func( $script );
+		} else {
+			echo $script;
+		}
```

Comment:
@mattheu not sure if this will be allowed in VIP? Any suggestions how to do it better?